model/container/registry.yaml

```
@@ -99,3 +99,23 @@ groups:
         brief: >
           Container labels, `<key>` being the label name, the value being the label value.
         examples: [ 'container.label.app=nginx' ]
+      - id: container.csi.plugin.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the CSI (Container Storage Interface) plugin used by the volume.
```

Shall we link to the CSI spec at https://github.com/container-storage-interface/spec for direct reference?

```suggestion
          The name of the CSI ([Container Storage Interface](https://github.com/container-storage-interface/spec)) plugin used by the volume.
```